### DISCUSSION TITLE
登录后显示白屏

### DISCUSSION THREAD

**Discussion by yang0009** [apolloconfig/apollo#4615] [Q&A]
登录界面登录不进去，显示白屏，偶尔甚至进不到主页面，一直停留在登录页面
前端报错信息如下：
angular.min.js?v=20210221023034:113 Error: [$resource:badcfg] http://errors.angularjs.org/1.5.1/$resource/badcfg?p0=find_favorites&p1=array&p2=object&p3=GET&p4=%2Ffavorites

**Comment by yang0009** [comment]
![image](https://user-images.githubusercontent.com/47616231/197721967-b233166c-463e-480f-91c1-6ab26d8384f2.png)

**Reply by yang0009** [reply]
版本：v1.0.0

**Comment by nobodyiam** [comment]
> Error in resource configuration for action `find_favorites`. Expected response to contain an array but got an object (Request: GET /favorites)

You may check the `Network` tab to see what's the result of `/favorites`.

**Reply by yang0009** [reply]
一开始登录不进去,没有这个/favorites 请求,然后一直重试登录,突然进来了,但是白屏
![image](https://user-images.githubusercontent.com/47616231/198214423-67f61e3f-8019-4c20-9026-c2568936f607.png)

**Reply by yang0009** [reply]
![image](https://user-images.githubusercontent.com/47616231/198214509-88bddbd0-3f62-4537-8a83-634f1e4a39c8.png)

**Comment by yang0009** [comment]
好像知道了,portal 副本数调成1 就好了,原来是2,这是定义好的吗

**Reply by nobodyiam** [reply]
可以看下 [session affinity](https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_241448-%e4%bb%a5ingress%e9%85%8d%e7%bd%aesession-affinity%e5%bd%a2%e5%bc%8f%e6%9a%b4%e9%9c%b2%e6%9c%8d%e5%8a%a1) 的配置。
如果是 1.9.0 及以上的版本，可以看下 [Portal 共享 session](https://www.apolloconfig.com/#/zh/development/portal-how-to-enable-session-store) 的配置。
